examples/connectivity/plot_adhd_covariance.py

```diff
@@ -8,11 +8,26 @@
  
```

Comment:
Maybe add more another half-setence on the used atlas because not knowing MSDL might complexify the example quite a bit.